packages/cli/commands/functions/deploy.js

```diff
@@ -30,6 +30,23 @@ const makeSpinner = (actionText, functionPath, accountIdentifier) => {
   );
 };
 
+const pollBuildStatus = (accountId, buildId) => {
```

Comment:
Nice. Is 1000 too fast?

Reply:
I set it to a faster poll because the build seems to happen pretty quickly in some cases. Although, this is a good point because the BE only has `PROCESSING`, `ERROR`, and `SUCCESS` states ATM, so we can dial this back. I'll go with `5000`